apps/src/templates/census2017/CensusForm.jsx

```diff
@@ -99,15 +116,49 @@ class CensusForm extends Component {
     });
   }
 
-  toggle(option) {
-    const selected = this.state.selected.slice(0);
+  togglePledge() {
+    this.setState({
+      submission: {
+        ...this.state.submission,
+        acceptedPledge: !this.state.submission.acceptedPledge
+      }
+    });
+  }
+
+  toggleHowMuchCS(option) {
+    const selected = this.state.selectedHowMuchCS.slice(0);
     if (selected.includes(option)) {
+      const newSelected = _.without(selected, option);
       this.setState({
-        selected: _.without(selected, option)
+        selectedHowMuchCS: newSelected,
```

Comment:
Now that we don't have any mutable operations, we don't need to duplicate our array on line 129, and this can just be `selectedHowMuchCS: _.without(this.state.selectedHowMuchCS, option)`

Below, we can go back to having lines 137 and 139 combined, since `.concat` returns the concatenated array (something `.push` did not do).